### ISSUE TITLE
After training a model, I find the training epoch seems not enough, so I reload it and try to finetune(retrain), but why the training loss seems like the beginning training loss (from scratch)?！ Is it because I save the model with '.model' , not '.t7' ??

### ISSUE THREAD

**Issue by brisker** [torch/torch7#887]
After training a model, I find the training epoch seems not enough, so I reload it and try to finetune(retrain), but why the training loss seems like the beginning training loss, just like I am training it from scratch?！ Is it because I save the model with '.model' , not '.t7' ??  I got really confused about this issue!

**Comment by soumith** [comment]
Could you post this to our google groups here: https://groups.google.com/forum/#!forum/torch7 , we only use this github issues for bug tracking the torch7 package.

**Comment by brisker** [comment]
@soumith I have done this. Could you please give some advice?
